Aleph Alpha's case study.

LukeMathWalker opened the pull request.

marcoow commented on src/cases/aleph-alpha.md:
```diff
@@ -1,13 +1,129 @@
+
+### Rust
+
+The entire data pipeline was built in Rust. That's the reason Aleph Alpha reached out to Mainmatter in the first place:
```
I'd mention Rust at the top where you talk about the need for performance?

LukeMathWalker replied:
Where are you referring to? The architecture section?

marcoow replied:
The challenge I think

marcoow reviewed (commented):
I think it's good. The only general comment I'd have is it might be a bit long and detailed and there's maybe potential for shortening a bit – I wouldn't overestimate people's willingness to read long success stories :)

I'd also have the copywriter we're working with for the service pages look over it – can't hurt. @SaeLLo should be able to make a connection

LukeMathWalker commented:
> The only general comment I'd have is it might be a bit long and detailed and there's maybe potential for shortening a bit – I wouldn't overestimate people's willingness to read long success stories

The way I think about it: to a certain audience, the content doesn't a matter at all—the only thing that matters is that we worked with company XYZ.  
To another audience (smaller, more tech savvy, perhaps in the very same AI field), adding a bit more detail helps to qualify us as experts.

sabine-devins commented on src/cases/aleph-alpha.md:
```diff
@@ -1,13 +1,131 @@
+
+## About Aleph Alpha
+
+Aleph Alpha is a German AI startup, a leader in the field of explainable and trustworthy sovereign AI.  
```
not sure what is meant by explainable? Maybe better word would be "accessible" or "understandable"

LukeMathWalker replied:
This is straight from their press release/company brand content, I ported it verbatim 🤷🏻‍♂️

LukeMathWalker commented on src/cases/aleph-alpha.md:
```diff
@@ -1,13 +1,131 @@
+
+We worked closely with Aleph Alpha's team to design a system that would satisfy all of these constraints, while 
+being **simple to operate** and **easy to reason about**.  
+
```
It's an engineering way of saying to refer to the quality/clarity of a chunk code/system—see https://softwareengineering.stackexchange.com/questions/351244/easy-to-reason-about-what-does-that-mean

LukeMathWalker commented:
Thanks a lot for the review @sabine-devins!
I think I've addressed all your comments (mostly accepted them as they are and clarified a few parts that you highlighted as confusing).

LukeMathWalker commented:
This is ready to go!

I can't seem to find the page in the preview though cc @yaigrie—could it have something to do with "Aleph Alpha" being two separate words?

marcoow commented:
This needs to be linked manually from the work page etc. @LukeMathWalker. There's a number of TODOs still – I think @yaigrie can take over:

- [ ] add `og:image`
- [ ] add some kind of visual also in the main content (maybe we can just use some of Aleph Alpha's press materials or so)
- [ ] add a large teaser for this on the `/work` page

Later we can also add this to the Rust landing page maybe and we should also add the quote we got (or will get?) from the to the Rust landing page.

LukeMathWalker commented:
The quote is already in the case study, so we can copy if from there and replace MVB's with it @marcoow

yaigrie commented:
@marcoow do you know where I can find those press images? Do we have a contact we can ask for such images?

marcoow commented:
@yaigrie ideally there's sth. on their website – we have contacts but they are super slow to respond so if we can go without them, that's eaiser

yaigrie commented:
@marcoow sadly there is no section on their website with images provided for press usage, I tried to google image search their images to see if they're using free stock, but got no results :(

We could try to use our own images, but it's a difficult topic to find images for
https://unsplash.com/photos/blue-and-white-water-wave-nbW-kaz2BlE
https://unsplash.com/photos/a-close-up-of-a-water-droplet-with-a-blue-background-kCrrUx7US04
https://unsplash.com/photos/a-very-tall-building-with-lots-of-windows-v0aq2kqDr7M (closeup?)
https://unsplash.com/photos/a-photo-taken-from-the-ground-looking-up-at-the-sky-bVO_LiMSVgw

marcoow commented:
Hm, too bad – maybe just a notebook with their website or sth.? Or maybe @LukeMathWalker has sth. – some code or graph showing the imported data or sth. (without revealing anything we can't reveal of course)…

LukeMathWalker commented:
Everything I have is in the aleph-alpha channel on Slack (mostly screenshots).

yaigrie commented:
@marcoow @LukeMathWalker I added a notebook showing some of the screenshots

marcoow reviewed (changes requested):
Let's remove the images we're not actually using. Also I think their logo is missing:

![Bildschirmfoto 2024-02-01 um 12 32 06](https://github.com/mainmatter/mainmatter.com/assets/1510/24545057-3d2e-4ae1-965c-cfcedb84289c)

Can we also feature the case study on the Rust landing page?

LukeMathWalker commented on src/cases/aleph-alpha.md:
```diff
@@ -1,13 +1,180 @@
+<div class="case-study__section">
+  <h2 class="case-study__heading h5">Outcome</h2>
+    <div class="case-study__text">
+        <p>We successfully assembled a high-quality multilingual dataset with more than <strong>4.5 trillion tokens</strong>. 
```
```suggestion
        <p>We successfully preprocessed <strong>4.5 trillion tokens</strong> to assemble a high-quality multilingual dataset. 
```

LukeMathWalker commented on src/cases/aleph-alpha.md:
```diff
@@ -1,13 +1,180 @@
 ---
 layout: case-study
 company: Aleph Alpha
+title: A large-scale data pipeline to feed AI models | Work
```
A few alternatives I considered:
- "Preprocessing trillions of tokens with Rust"
- "Large models need large datasets: preprocessing trillions of tokens with Rust"

marcoow replied:
I think the first one is good – bit more catchy than the latter :)

LukeMathWalker replied:
Yup, that's my favourite as well.

LukeMathWalker commented:
The graphs in the notebook image must be redacted @yaigrie—no numbers on either axis, no titles.

yaigrie commented:
@marcoow if we soon merge both case studies, this one and rail europe, which one should be at the top of the work page?
I think I resolved all issues now :)

marcoow commented:
let's put this one at the top

marcoow reviewed (approved):
🙌
